src/spec/payid-protocol.md

```diff
@@ -218,20 +218,20 @@ This specification defines the PayID protocol - an application-layer protocol, w
 ## Meaning of Media Type application/* + json
   `*` may represent different payment-networks and environments. In this document, we propose standards with the media types specific to XRP, ILP, and ACH payment networks. We also propose media types that return all addresses across all payment networks. Other payment networks MUST establish standard media types for their networks at IANA.
 
-  * Accept: application/payid-json
+  * Accept: application/payid+json
   Returns all payment account(s) information corresponding to the requested PayID URI
 
   * Accept: application/xrpl-mainnet+json
 
-  Returns XRPL mainnet XAddresses
+  Returns XRPL mainnet classic addresses (and tags)
```

Comment:
We landed on classic XRPL addresses, not X-Addresses.

Reply:
I agree with this change -- although I found myself publishing both formats because they each have their usages. Is there any reason not to keep XAddress as an "optional" field?

E.g., try `curl https://fluid.money/pay`

Reply:
> although I found myself publishing both formats because they each have their usages

What use case do you see for a PayID server serving up an X-Address? In my mind, X-Address is entirely a display issue / client-side concern.

> Is there any reason not to keep XAddress as an "optional" field?

@keefertaylor might be able to explain this more eloquently than me, but right now we basically have two object shapes `CryptoAddressDetails | FiatAddressDetails`.

If you add `xAddress` as an optional field to XRPL PayID addresses, now you have 3 object shapes: `XrplAddressDetails | CryptoAddressDetails | FiatAddressDetails`. There's no real reason to special case XRPL, and if we break away from `CryptoAddressDetails`, it's possible other cryptocurrencies would too, which makes the client-side logic more difficult to implement.F

Also, in my mind, it's never a great idea to have two sources of truth. What if your `xAddress` is not actually equivalent to your `address + tag`?

Reply:
Yeah, those are convincing rationale. Thanks!